### DISCUSSION TITLE
talk/

### DISCUSSION THREAD

**Discussion by giscus** [GZTimeWalker/blog-comments#3] [Announcements]
# talk/

Walking on the Time Axis.

https://blog.gztime.cc/talk/

**Comment by GZTimeWalker** [comment]
由于说说之前也是基于 Leancloud 的……想了想干脆弄个空白页吧，然后把评论当作发送说说的地方。
**新年快乐！！**

**Comment by ZexuanChen** [comment]
你好，我想问一下我用butterfly主题配置的网站别人都加载不出来该怎么办:cry:，感觉是网络原因

**Reply by GZTimeWalker** [reply]
啊，才发现是校友，别人加载不出来你的网站大概率是你没部署好了，相关问题的具体解决方案或许你可以找到我的其他联系方式联系我？
